content/en/docs/security/_index.md

```diff
@@ -2,3 +2,41 @@
 title: Security
 weight: 970
 ---
+
+Learn how the OpenTelemetry project discloses vulnerabilities and responds to
+incidents. Find out how to ensure your observability data is collected and
+transmitted in a secure manner.
```

Comment:
should we move that into the frontmatter `description`, it reads like one

Reply:
It might be a little long for the `description`. Let me take another shot at making the wording more intro-like.

Reply:
I rewrote this in f099457.